driver-adapters/neon-vercel-nextjs/prisma/schema.prisma

```diff
@@ -0,0 +1,15 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DRIVER_ADAPTERS_NEON_VERCEL_NEXTJS_DATABASE_URL")
```

Comment:
It might be good idea to set this to an invalid value to make sure it is not being used under the hood instead of the _actual_ driver with the defined-in-code value.